packages/macros/src/babel-plugin-cache-busting.ts

```diff
@@ -0,0 +1,3 @@
+export default function makePlugin(): any {
+  return {};
+}
```

Comment:
This seems like exactly what was added in https://github.com/embroider-build/embroider/pull/841. 


Can we just use that plugin here (we might need to move it to `@embroider/utils`, but that seems fine)?

https://github.com/embroider-build/embroider/blob/7165f06bb6027e82c61fe0ab107e54f9f0c6ff8c/packages/core/src/babel-plugin-cache-busting.ts#L1-L11

Reply:
Will move to `@embroider/shared-internals`